elements/pfe-icon/demo/pfe-icon.story.js

```diff
@@ -1,85 +1,11 @@
 import { storiesOf } from "@storybook/polymer";
-import {
```

Comment:
@mwcz I would hope we don't need to regress the story this much - can I jump in and help with re-implementing the attributes for styling?

Reply:
I haven't worked on the story at all yet.  I recreated the element from scratch so the previous icon's story was removed.  I've been pulling things back in from the previous implementation piece by piece, and haven't gotten to the story yet.

Reply:
Oh I see!  That's great, let me know if I can help with anything.